src/rules.js

```diff
@@ -173,6 +173,10 @@ function getPluginPackageName(pluginName) {
     }
 
     if (pluginName.startsWith('@')) {
+        if (pluginName.includes('/')) {
+            const [scope, plugin] = pluginName.split('/');
+            return `${scope}/${eslintPluginPrefix}-${plugin}`;
+        }
```

Comment:
The use case this PR addresses is already covered in this function (lines 170 - 173).  These lines are not needed.

```suggestion
```